src/resources/Sources/SourcesMappings/SourcesMappingsInterfaces.ts

```diff
@@ -0,0 +1,43 @@
+import {FieldTypes, MappingModelKind} from '../../Enums';
+
+export interface GetSourceMappingParams {
+    organizationId?: string;
+}
```

Comment:
Where is this used?

Reply:
This ended up not being used and has been removed :+1: